scout/server/blueprints/variants/controllers.py

```diff
@@ -225,6 +225,46 @@ def sv_variants(store, institute_obj, case_obj, variants_query, variant_count, p
     return {"variants": variants, "more_variants": more_variants}
 
 
+def mei_variants(
```

Comment:
As usual, the variants controllers (and views) should be refactored, but it will be a lot to do it in this PR.